ui/src/services/NodeVolumesUtils.js

```diff
@@ -223,21 +231,26 @@ export const getVolumeListData = createSelector(
         volumeHealth = STATUS_HEALTH;
       }
 
-      const instanceIP = nodeList.find(
+      const instanceIP = nodeList?.find(
         (node) => node.name === volume?.spec?.nodeName,
       )?.internalIP;
 
+      const volumeCurrentLatency = volumeLatencyCurrent?.find(
+        (vLV) =>
+          vLV.metric.device === volume?.status?.deviceName &&
+          vLV.metric.instance === instanceIP + PORT_NUMBER_PROMETHEUS,
```

Comment:
Shouldn't it be:
```suggestion
          vLV.metric.instance === `${instanceIP}:${PORT_NUMBER_PROMETHEUS}`,
```
?

Reply:
correct! It's more understandable!

Reply:
But it doesn't mean the same thing... So is `PORT_NUMBER_PROMETHEUS` actually a string containing `:9090`?

Reply:
yes.. originally the value of `PORT_NUMBER_PROMETHEUS` is `:9090` 
but yeah, it doesn't make any sense.